spotify_tensorflow/dataset.py

```diff
@@ -242,23 +269,27 @@ def __iter__(self):
                 yield self.buff
 
             def _get_buff_size(self):
+                # type: () -> int
                 if self.buff is None:
                     return 0
                 else:
                     return len(self.buff[list(self.buff.keys())[0]])
 
             def _append(self, v1, v2):
+                # type: (np.ndarray, np.ndarray) -> np.ndarray
                 if type(v1) is np.ndarray:
-                    if(v1.ndim == 1):
+                    if v1.ndim == 1:
                         return np.append(v1, v2)
-                    elif(v1.ndim == 2):
+                    elif v1.ndim == 2:
                         return np.vstack([v1, v2])
                     else:
                         raise ValueError("Only 1 or 2 dimensional features are supported")
                 else:
+                    # TODO: what case does this code path support?
```

Comment:
@richwhitjr @yonromai ?

Reply:
Yeah I think it was the fall through case for other types of object for example if we got strings etc working.  The thought was to fall back to normal append and hope it works.  Maybe raise an error and add the new types as we see them?

Reply:
@richwhitjr i see, we should probably add a comment here - could you pls submit a PR for this?

Reply:
Sure will do.